atd-vzd/schema/afd__incidents.sql

```diff
@@ -0,0 +1,65 @@
+DROP TABLE public.afd__incidents;
+
+-- Table Definition
+CREATE TABLE "public"."afd__incidents" (
+    id serial primary key,
+    incident_number integer,
+    crash_id integer default null,
+    unparsed_ems_incident_number text default null,
+    ems_incident_numbers integer[] default null,
+    call_datetime timestamp default null,
+    calendar_year text default null,
+    jurisdiction text default null,
+    address text default null,
+    problem text default null,
+    flagged_incs text default null,
+    geometry geometry(Point,4326) default null,
+    austin_full_purpose bool default null,
+    location_id text default null,
+    latitude float8 default null,
+    longitude float8 default null,
+    ems_incident_number_1 integer default null,
+    ems_incident_number_2 integer default null,
+    call_date date default null,
+    call_time time default null 
+);
+
+CREATE OR REPLACE FUNCTION afd_incidents_trigger()
+  RETURNS trigger AS
+$$
+BEGIN
+  update afd__incidents set
```

Comment:
When the trigger runs, it essentially does so only to have this query execute on the row in question.